spec/src/main/asciidoc/integration.adoc

```diff
@@ -0,0 +1,79 @@
+//
+// Copyright (c) 2023 Contributors to the Eclipse Foundation
+//
+// See the NOTICE file(s) distributed with this work for additional
+// information regarding copyright ownership.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+== SDK integration
+
+Implementations SHALL provide integration by appropriately configuring compatible OpenTelemetry SDK for application runtime.
+Regardless of signal type being considered -- traces or metrics -- following requirements SHALL be met:
+
+[[sec:enabling-opentelemetry-support]]
+=== Enabling OpenTelemetry support
+By default, MicroProfile Telemetry is deactivated.
+
+In order to enable any of aspects of integration, the configuration `otel.sdk.disabled=false` MUST be specified in any of the configuration sources available via MicroProfile Config.
```

Comment:
This should be a recommendation only. Implementations can opt to have  MicroProfile Telemetry as active by default. There is not technical reason preventing that.

Reply:
I agree, that implementation may choose other means of enabling MP Telemetry. 

But in order to explicitly signal "this application requires Telemetry" such as in TCK test, you must provide this property and runtime must honor that.

Reply:
From a practical point of view, MP Telemetry will require configuration in almost all instances, which I think is why we specify it being disabled by default.

If it was enabled by default, then I'd expect that a new user of any runtime which supports MP Telemetry would be immediately confused by the errors in the log saying it couldn't connect to the default OTLP server.

I wouldn't be against making it a recommendation - in some cases it might make sense e.g. if the implementation is supplied as a library to be packaged in an application, the user is opting in to using MP Telemetry by including the library so the additional config is less important. However, we should add some text explaining why we're recommending it, so that implementers can make an informed decision.

Reply:
This is just to settle the behavior, not the default value.